MAINTAINING.md

```diff
@@ -0,0 +1,147 @@
+# RIOT Review Guidelines
+
+The items presented in the following list should be checked for each PR review,
+in this order. This list is however not exhaustive and defines a rather minimal
+baseline on things that should be checked.
+
+Note: Any of the items in this list might be skipped, if **good** reasons are presented.
+
+Also note: This reviewing order is meant to maximize efficiency and minimize overhead, since failing in step n makes steps n+x obsolete. Maintainers should be aware that starting a review with 4. (Code Style) not only wasts their own time, but also the time of the author
+
+## 1. - Review the fundaments
+
+1.1 Does the reasoning for this PR make sense?
+1.2 Do the concepts used by the PR make sense? 
+1.3 Does the PR break with existing concepts?
+1.4 Is the structure of the PR itself valid?
```

Comment:
Having not reviewed that much yet, I'm not sure what this means. Does this mean e.g. not combining completely unrelated contributions, or half-contributions, and that kind of thing?